src/common/Experiments/Experiments-Live/Experiments.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
```

Comment:
It would probably be a good idea to put a note in this file clarifying that the dependencies required to build it are only available in the official build pipeline and are internal to Microsoft, but that the project is not required to build a test version of the application.